sympy/functions/elementary/tests/test_exponential.py

```diff
@@ -167,6 +167,8 @@ def test_log_values():
 
     assert log(oo*I) == oo
     assert log(-oo*I) == oo
+    assert log(0, 2) == zoo
+    assert log(0, 5) == zoo
```

Comment:
Is this related to multiplicity? I'd have thought that this is simply what you return for "it's a complex pole around zero".